marketplace-service/src/main/java/com/axonivy/market/github/service/impl/GHAxonIvyProductRepoServiceImpl.java

```diff
@@ -170,6 +177,19 @@ public ProductModuleContent getReadmeAndProductContentsFromTag(Product product,
     return productModuleContent;
   }
 
+  /**
+   * MARP-810: Sabine requires that content in other languages, which has not been translated, be left empty and replaced with English content.
+   */
+  public Map<String, String> replaceEmptyContentsWithEnContent(Map<String, String> map) {
+    String enValue = map.get(Language.EN.getValue());
+    for (Map.Entry<String, String> entry : map.entrySet()) {
+      if (entry.getValue().isEmpty() && !entry.getKey().equals(Language.EN.getValue())) {
```

Comment:
1. use StringUtils.isBlank() instead of using isEmpty()
2. I think the 2nd condition is not nessessory